src/ResourceManager/EventHub/Commands.EventHub/Cmdlets/Namespace/NewAzureEventHubNamespace.cs

```diff
@@ -64,43 +83,100 @@ public class NewAzureEventHubNamespace : AzureEventHubsCmdletBase
         /// </summary>
         [Parameter(
           Mandatory = false,
+            Position = 3,
           ValueFromPipelineByPropertyName = true,
-          HelpMessage = "Namespace Sku Name.")]
+          HelpMessage = "Namespace Sku Name.",
+          ParameterSetName = NamespaceParameterSet
+            )]
         [ValidateSet(SKU.Basic,
           SKU.Standard,
           SKU.Premium,
           IgnoreCase = true)]
+        [Parameter(
+          Mandatory = false,
+            Position = 3,
+          ValueFromPipelineByPropertyName = true,
+          HelpMessage = "Namespace Sku Name.",
+          ParameterSetName = AutoInflateParameterSet
+            )]
         public string SkuName { get; set; }
 
-
         /// <summary>
         /// The eventhub throughput units.
         /// </summary>
         [Parameter(
           Mandatory = false,
+            Position = 4,
+          ValueFromPipelineByPropertyName = true,
+          HelpMessage = "The eventhub throughput units.",
+          ParameterSetName = NamespaceParameterSet
+            )]
+        [Parameter(Mandatory = false,
+            Position = 4,
           ValueFromPipelineByPropertyName = true,
-          HelpMessage = "The eventhub throughput units.")]
+          HelpMessage = "The eventhub throughput units.",
+          ParameterSetName = AutoInflateParameterSet)]
         public int? SkuCapacity { get; set; }
         
         /// <summary>
         /// Hashtables which represents resource Tags.
         /// </summary>
         [Parameter(
             Mandatory = false,
+            Position = 5,
+            ValueFromPipelineByPropertyName = true,
+            HelpMessage = "Hashtables which represents resource Tags.",
+            ParameterSetName = NamespaceParameterSet
+            )]
+        [Parameter(
+            Mandatory = false,
+            Position = 5,
             ValueFromPipelineByPropertyName = true,
-            HelpMessage = "Hashtables which represents resource Tags.")]
+            HelpMessage = "Hashtables which represents resource Tags.",
+            ParameterSetName = AutoInflateParameterSet
+            )]
         public Hashtable Tag { get; set; }
 
         /// <summary>
+        /// Indicates whether AutoInflate is enabled.
+        /// </summary>
+        [Parameter(
+            Mandatory = true,
+            Position = 6,
```

Comment:
@v-Ajnava `SwitchParameters` do not need positions nor do they accept a value from the pipeline